docs/.vuepress/config.js

```diff
@@ -155,9 +155,26 @@ module.exports = {
     lastUpdated: 'Last Updated',
   },
   dest: '_site',
-  plugins: ['@vuepress/google-analytics'],
+  plugins: [
+    '@vuepress/google-analytics',
+    [
+      '@vuepress/pwa',
+      {
+        serviceWorker: true,
+        updatePopup: false,
```

Comment:
so how will new content happen then?

Reply:
It uses skipwaiting, see: https://github.com/open-wc/open-wc/pull/1053#issuecomment-559741070